ray-operator/config/crd/patches/cainjection_in_rayservice_rayservices.yaml

```diff
@@ -0,0 +1,7 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
```

Comment:
Could you more about what this file is? Was it autogenerated?

Reply:
Autogenerated